types/hapi/test/server/server-cache-provision.ts

```diff
@@ -0,0 +1,21 @@
+// https://github.com/hapijs/hapi/blob/master/API.md#-await-servercacheprovisionoptions
+import {Server} from "hapi";
+import * as catbox from "catbox";
+
+const server = new Server({
+    port: 8000,
+});
+server.initialize();
+server.cache.provision({engine: require('catbox-memory'), name: 'countries' });
+
+const cache:catbox.Policy = server.cache({segment: 'countries', cache: 'countries', expiresIn: 60 * 60 * 1000 });
+cache.set('norway', 'oslo', 10 * 1000, null);
+const value = cache.get('norway', null);
+
+// TODO Not working. It needs review.
```

Comment:
?

Reply:
Sorry. It was an old comment. @BorntraegerMarc told me to take it out but I forgot. Commit 3fe936d